frontend/src/components/Button/Button.scss

```diff
@@ -1,3 +1,6 @@
+@import "stylebase/variables";
+@import "stylebase/utils";
```

Comment:
Index.scss should only have global styles. The button styles should not be global (and therefore should not be in index.scss) this keeps the styles modular and makes the user only download the styles they need for the particular page they are on right now.

Reply:
Yeah true. That's the reason I brought it up because many of the stylings were imported into the index.scss. We need to stick to a method I guess. What I saw gave me the courage to insert all my CSS files into `index.scss` file.

Importing it twice will result to duplicate. A perfect example is an image below.

For Example:
```
@import "stylebase/variables";
@import "stylebase/type";
@import "stylebase/utils";
@import "stylebase/global";

@import "components/Button/Button";
@import "components/Map/Map";
```

![duplicate-styling](https://user-images.githubusercontent.com/31921958/78301230-b2fc9880-7530-11ea-95cb-b317b24e7902.png)